code/WorkInProgress/laundry.dm

```diff
@@ -70,7 +70,7 @@
 					C.stains = list("damp")
 					C.UpdateName()
 				I.clean_forensic()
-			if (src.occupant)
+			if (src.occupant && ishuman(src.occupant))
```

Comment:
Note that you also could've done `&& istype(H)` (not saying you should change it, just pointing out an alternative).